internal/cli/atlas/whitelist/describe.go

```diff
@@ -25,6 +25,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const describeTemplate = `IP ADDRESS	SECURITY GROUP
```

Comment:
can we align this with list? `CIDR BLOCK`?

Reply:
Whoops yes they were meant to be the same